redis_opentracing/tracing.py

```diff
@@ -1,3 +1,4 @@
+from builtins import str
```

Comment:
Is this the change that fixes unicode in statements? Otherwise I see nothing related to this ;)